src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -225,10 +225,10 @@ public async ValueTask<Variant> GetVariantAsync(string feature, CancellationToke
         /// Gets the assigned variant for a specific feature.
```

Comment:
In the `public async ValueTask<Variant> GetVariantAsync(string feature, CancellationToken cancellationToken = default)` method,

I can't see any reason that we don't call `await EvaluateFeature<object>(feature, context: null, useContext: false, cancellationToken);` instead of `await EvaluateFeature<TargetingContext>(feature, context: null, useContext: false, cancellationToken);` similarly to how we handle non-contextual IsEnabledAsync.